lib/potassium/version.rb

```diff
@@ -1,6 +1,6 @@
 module Potassium
-  VERSION = "3.0.0"
+  VERSION = "5.1.0"
   RUBY_VERSION = "2.3.1"
-  RAILS_VERSION = "~> 5.0.2"
+  RAILS_VERSION = "~> 5.1.0.rc1"
```

Comment:
Freeze mutable objects assigned to constants.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.